R/utils-shared.R

```diff
@@ -35,6 +35,70 @@ startRender <- function(data, output, author, reportName){
 
 ############ used in dvhydrograph-data, fiveyeargwsum-data, uvhydrograph-data ############ 
 
+getGroundWaterLevels<- function(ts, ...){
+  y <- as.numeric(ts$gwlevel[['groundWaterLevel']])
+  x <- ts$gwlevel[['dateString']]
+  time = as.POSIXct(strptime(x, "%Y%m%d"))
+  month <- format(time, format = "%y%m") #for subsetting later by month
+  return(data.frame(time=time, value=y, month=month, stringsAsFactors = FALSE))
+}
```

Comment:
only other comment: should we have a NULL check on this? getWaterQualityMeasurements has one, but some others don't. Or should that be addressed when we make that common null checking function?